### ISSUE TITLE
Prefer const returning false positives

### ISSUE THREAD

**Issue by xscheiner** [palantir/tslint#2309]
### Bug Report

- __TSLint version__: 4.5.1
- __TypeScript version__: 2.1.0
- __Running TSLint via__: (pick one) gulp-tslint

#### TypeScript code being linted

```ts
const value: string = el.value;
```

with `tslint.json` configuration:

```json
{
  "extends": "tslint:latest",
  "rules": {
    "no-namespace": [
      false
    ],
    "member-access": [
      false
    ],
    "object-literal-shorthand": false,
    "trailing-comma": [
      false
    ],
    "no-angle-bracket-type-assertion": false,
    "object-literal-sort-keys": false,
    "whitespace": [
      false
    ],
    "no-consecutive-blank-lines": [
      true
    ],
    "no-console": [
      false
    ],
    "no-string-literal": false,
    "ban-types": false,
    "only-arrow-functions": [
      false
    ]
  }
}
```

#### Actual behavior
```
src/main/class-code/code-entry-signup.ts[197, 11]: Identifier 'value' is never reassigned; use 'const' instead of 'let'.
```
#### Expected behavior
Code already uses const, no error should show

**Comment by ajafff** [comment]
Can you provide more code to reproduce the bug?
Maybe the variable is shadowed or something ...
With the provided code snipped I can only guess.

**Comment by JoshuaKGoldberg** [comment]
👏 @xscheiner closing this issue for housekeeping purposes - if this is still causing problems on your end, please do post some source code we can take a look at!
